bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
It's okay, I already know it from having those subpackages :) I was just wondering if this usage is leaking into the open wild, in which case it may be good to make an announcement to stop it.